### PR TITLE
feat: added restart button for containers

### PR DESCRIPTION
**Pull request by SashaGoncharov19**
https://github.com/Dokploy/dokploy/issues/411

### CODE REVIEW

File: apps/dokploy/components/dashboard/compose/monitoring/show.tsx [hunk 1]
```diff
@@ -1,86 +1,115 @@
...
+            <Button
+              onClick={async () => {
+                toast.success(`Restarting container ${containerId}`);
+                await restart({ containerId }).then(() => {
+                  toast.success("Container restarted");
+                });
+              }}
+            >
+              Restart
+            </Button>
```
Comment (Siumauricio) [inline apps/dokploy/components/dashboard/compose/monitoring/show.tsx]: add isLoading prop when the mutation is executing

File: apps/dokploy/components/dashboard/compose/monitoring/show.tsx [hunk 2]
```diff
@@ -1,86 +1,115 @@
...
+                      {container.state}
+                    </SelectItem>
+                  ))}
+                  <SelectLabel>Containers ({data?.length})</SelectLabel>
+                </SelectGroup>
+              </SelectContent>
+            </Select>
+            <Button
+              onClick={async () => {
+                toast.success(`Restarting container ${containerId}`);
```
Comment (Siumauricio) [inline apps/dokploy/components/dashboard/compose/monitoring/show.tsx]: instead of containerId, can we show the container name?

**Review by Siumauricio — CHANGES_REQUESTED** [review]

File: apps/dokploy/components/dashboard/compose/monitoring/show.tsx (hunk 2 above)
Comment (SashaGoncharov19) [reply apps/dokploy/components/dashboard/compose/monitoring/show.tsx]: yep, it will be better

**Review by Siumauricio — APPROVED** [review]

**Comment by Siumauricio** [comment]
@SashaGoncharov19 Run pnpm run check to fix lint issues

**Pull request merged**